test/products/ProductMapperTest.js

```diff
@@ -189,6 +189,21 @@ describe('Magento ProductMapper', () => {
             assert.strictEqual(response.sku, 'testSimpleProduct');
         });
 
+        it('maps attributes correctly with JavaScript object of target attributes', () => {
+            productMapper.attributes = ["custom_string_attribute", "media_gallery_entries"];
```

Comment:
Can you please replace `custom_string_attribute` with `name` and remove this extra attribute from the sample data? (we don't need an extra attribute here)